### PR TITLE
add Japanese translation of pod-priority-preemption.md

### LINKED ISSUE #25052: Translate concepts/configuration/pod-priority-preemption/ into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate [concepts/configuration/pod-priority-preemption/](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/) into Japanese

**Why is this needed**

This page has not been translated.

**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by translucens**
Close #25052

### CODE REVIEW

File: content/ja/docs/concepts/configuration/pod-priority-preemption.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+値は`value`フィールドで指定し、必須です。
+値が大きいほど、高い優先度を示します。
+PriorityClassオブジェクトの名称は[DNSサブドメイン名](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)として適切であり、かつ`system-`から始まらないことが必要です。
+
+PriorityClassオブジェクトは10億以下の任意の32ビットの整数値を持つことができます。
+それよりも大きな値は通常はプリエンプトや追い出すべきではない重要なシステム用のPodのために予約されています。
+クラスターの管理者は割り当てたい優先度に対して、PriorityClassオブジェクトを1つずつ作成すべきです。
+
+PriorityClassは任意でフィールド`globalDefault`と`description`を設定可能です。
+`globalDefault`フィールドは`priorityClassName`が指定されないPodはこのPriorityClassを使うべきであることを示します。`globalDefault`が設定されたPriorityClassはシステムで一つのみ存在可能です。`globalDefault`が設定されたPriorityClassが存在しない場合は、`priorityClassName`が設定されていないPodの優先度は0に設定されます。
```
Comment (kakts) [inline content/ja/docs/concepts/configuration/pod-priority-preemption.md:59]:
```suggestion
`globalDefault`フィールドは`priorityClassName`が指定されないPodはこのPriorityClassを使うべきであることを示します。`globalDefault`がtrueに設定されたPriorityClassはシステムで一つのみ存在可能です。`globalDefault`が設定されたPriorityClassが存在しない場合は、`priorityClassName`が設定されていないPodの優先度は0に設定されます。
```

>  Only one PriorityClass with globalDefault set to true can exist in the system.

File: content/ja/docs/concepts/configuration/pod-priority-preemption.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+
+`description`フィールドは任意の文字列です。クラスターの利用者に対して、PriorityClassをどのような時に使うべきか示すことを意図しています。
+
+### PodPriorityと既存のクラスターに関する注意
+
+-   もし既存のクラスターをこの機能がない状態でアップグレードすると、既存のPodの優先度は実質的に0になります。
+
+-   `globalDefault`が`true`に設定されたPriorityClassを追加しても、既存のPodの優先度は変わりません。PriorityClassのそのような値は、PriorityClassが追加された以後に作成されたPodのみに適用されます。
+
+-   PriorityClassを削除した場合、既存のPodは削除されたPriorityClassの名称を引き続き使用しますが、削除されたPriorityClassの名称を使うPodをそれ以上作成することはできなくなります。
```
Comment (kakts) [inline content/ja/docs/concepts/configuration/pod-priority-preemption.md:69]:
```suggestion
-   PriorityClassを削除した場合、削除されたPriorityClassの名前を使用する既存のPodは変更されませんが、削除されたPriorityClassの名前を使うPodをそれ以上作成することはできなくなります。
```

> If you delete a PriorityClass, existing Pods that use the name of the deleted PriorityClass remain unchanged, but you cannot create more Pods that use the name of the deleted PriorityClass.

File: content/ja/docs/concepts/configuration/pod-priority-preemption.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+スケジューリングされるのを待つ非プリエンプトのPodは、リソースが十分に利用可能になるまでスケジューリングキューに残ります。
+非プリエンプトのPodは、他のPodと同様に、スケジューラーのバックオフの対象になります。これは、スケジューラーがPodをスケジューリングしようと試みたものの失敗した場合、低い頻度で再試行するようにして、より優先度の低いPodが先にスケジューリングされることを許します。
+
+非プリエンプトのPodは、他の優先度の高いPodにプリエンプトされる可能性はあります。
+
+`PreemptionPolicy`はデフォルトでは`PreemptLowerPriority`に設定されており、これが設定されているPodは優先度の低いPodをプリエンプトすることを許容します。これは既存のデフォルトの挙動です。
+`PreemptionPolicy`を`Never`に設定すると、これが設定されたPodはプリエンプトを行わないようになります。
+
+ユースケースの例として、データサイエンスの処理を挙げます。
+ユーザーは他の処理よりも優先度を高くしたいジョブを追加できますが、そのとき既存の実行中のPodの処理結果をプリエンプトによって捨てさせたくはありません。
```
Comment (kakts) [inline content/ja/docs/concepts/configuration/pod-priority-preemption.md:97]:
```suggestion
ユーザーは他の処理よりも優先度を高くしたいジョブを追加できますが、そのとき既存の実行中のPodの処理結果をプリエンプトによって破棄させたくはありません。
```
翻訳していただいた文でも問題ありませんが、破棄という文言が良いかと思いました!

> A user may submit a job that they want to be prioritized above other workloads, but do not wish to discard existing work by preempting running pods

File: content/ja/docs/concepts/configuration/pod-priority-preemption.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+
+## プリエンプション
+
+Podが作成されると、スケジューリング待ちのキューに入り待機状態になります。スケジューラーはキューからPodを取り出し、ノードへのスケジューリングを試みます。Podに指定された条件を全て満たすノードが見つからない場合は、待機状態のPodのためにプリエンプションロジックが発動します。待機状態のPodをPと呼ぶことにしましょう。プリエンプションロジックはPよりも優先度の低いPodを一つ以上追い出せばPをスケジューリングできるようになるノードを探します。そのようなノードがあれば、優先度の低いPodはノードから追い出されます。Podが追い出された後に、Pはノードへスケジューリング可能になります。
+
+### ユーザーへ開示される情報
+
+Pod PがノードNのPodをプリエンプトした場合、ノードNの名称がPのステータスの`nominatedNodeName`フィールドに設定されます。このフィールドはスケジューラーがPod Pのために予約しているリソースの追跡を助け、ユーザーにクラスターにおけるプリエンプトに関する情報を与えます。
+
+Pod Pは「指名したノード」へスケジューリングされる必要はないことに注意してください。Podがプリエンプトされると、そのPodは猶予期間を得ます。スケジューラーがPodの終了を待つ間に他のノードが利用可能になると、スケジューラーは他のノードをPod Pのスケジューリング先にします。この結果、Podの`nominatedNodeName`と`nodeName`は必ずしも一致しません。また、スケジューラーがノードNのPodをプリエンプトさせた後に、Pod Pよりも優先度の高いPodが来た場合、スケジューラーはノードNをその新しい優先度の高いPodへ与えます。このような場合は、スケジューラーはPod Pの`nominatedNodeName`を消去します。これによって、スケジューラーはPod Pが他のノードのPodをプリエンプトさせられるようにします。
```
Comment (kakts) [inline content/ja/docs/concepts/configuration/pod-priority-preemption.md:146]:
```suggestion
Pod Pは必ずしも「指名したノード」へスケジューリングされないことに注意してください。Podがプリエンプトされると、そのPodは終了までの猶予期間を得ます。スケジューラーがPodの終了を待つ間に他のノードが利用可能になると、スケジューラーは他のノードをPod Pのスケジューリング先にします。この結果、Podの`nominatedNodeName`と`nodeName`は必ずしも一致しません。また、スケジューラーがノードNのPodをプリエンプトさせた後に、Pod Pよりも優先度の高いPodが来た場合、スケジューラーはノードNをその新しい優先度の高いPodへ与えます。このような場合は、スケジューラーはPod Pの`nominatedNodeName`を消去します。これによって、スケジューラーはPod Pが他のノードのPodをプリエンプトさせられるようにします。
```

> Please note that Pod P is not necessarily scheduled to the "nominated Node".



終了までの猶予期間
>  their graceful termination period.
gracefulをうまく言い換えれませんが、terminationを入れるとこういう感じでしょうか？

**Review by kakts — CHANGES_REQUESTED** [review]
L147の途中までみました。いくつかコメントしましたので、ご確認お願いいたします。

**Comment by kakts** [comment]
一通り確認しました

File: content/ja/docs/concepts/configuration/pod-priority-preemption.md [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+kind: PriorityClass
+metadata:
+  name: high-priority-nonpreempting
+value: 1000000
+preemptionPolicy: Never
+globalDefault: false
+description: "この優先度クラスは他のPodをプリエンプトさせません。"
+```
+
+## Podの優先度
```
Comment (sftim) [inline content/ja/docs/concepts/configuration/pod-priority-preemption.md:113]:
```suggestion
## Podの優先度 {#pod-priority}
```

File: content/ja/docs/concepts/configuration/pod-priority-preemption.md [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+kubeletは追い出すPodの順位付けを次の順で行います。枯渇したリソースを要求以上に使用しているか、優先度、枯渇したリソースの消費量の複数のPodの要求に対する相対値。
+詳細は[エンドユーザーのPodの追い出し](/docs/tasks/administer-cluster/out-of-resource/#evicting-end-user-pods)を参照してください。
+
+
+kubeletによるリソース不足時のPodの追い出しでは、リソースの消費が要求を超えないPodは追い出されません。優先度の低いPodのリソースの利用量がその要求を超えていなければ、追い出されることはありません。より優先度が高く、要求を超えてリソースを使用しているPodが追い出されます。
+
+
+## {{% heading "whatsnext" %}}
+
+* PriorityClassと関連付けてResourceQuotaを使用することに関して [デフォルトで優先度クラスの消費を制限する](/ja/docs/concepts/policy/resource-quotas/#limit-priority-class-consumption-by-default)
```
Comment (sftim) [inline content/ja/docs/concepts/configuration/pod-priority-preemption.md:240]: nit:
```suggestion
* PriorityClassと関連付けてResourceQuotaを使用することに関して [デフォルトで優先度クラスの消費を制限する](/ja/docs/concepts/policy/resource-quotas/#limit-priority-class-consumption-by-default/)
```

**Review by sftim — COMMENTED** [review]
Two small suggestions

File: content/ja/docs/concepts/configuration/pod-priority-preemption.md (hunk 6 above)
Comment (translucens) [reply content/ja/docs/concepts/configuration/pod-priority-preemption.md]: linked document has another name's anchor so I changed its name in the URL.

File: content/ja/docs/concepts/configuration/pod-priority-preemption.md [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+これらは汎用のクラスであり、[重要なコンポーネントが常に最初にスケジュールされることを保証する](/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/)ために使われます。
+{{< /note >}}
+
+## PriorityClass
+
+PriorityClassはnamespaceによらないオブジェクトで、優先度クラスの名称から優先度を表す整数値への対応を定義します。
+PriorityClassオブジェクトのメタデータの`name`フィールドにて名称を指定します。
+値は`value`フィールドで指定し、必須です。
+値が大きいほど、高い優先度を示します。
+PriorityClassオブジェクトの名称は[DNSサブドメイン名](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)として適切であり、かつ`system-`から始まらないことが必要です。
```
Comment (nasa9084) [inline content/ja/docs/concepts/configuration/pod-priority-preemption.md:52]:
```suggestion
PriorityClassオブジェクトの名称は[DNSサブドメイン名](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)として適切であり、かつ`system-`から始まってはいけません。
```

File: content/ja/docs/concepts/configuration/pod-priority-preemption.md [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+これらは汎用のクラスであり、[重要なコンポーネントが常に最初にスケジュールされることを保証する](/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/)ために使われます。
+{{< /note >}}
+
+## PriorityClass
+
+PriorityClassはnamespaceによらないオブジェクトで、優先度クラスの名称から優先度を表す整数値への対応を定義します。
+PriorityClassオブジェクトのメタデータの`name`フィールドにて名称を指定します。
+値は`value`フィールドで指定し、必須です。
+値が大きいほど、高い優先度を示します。
+PriorityClassオブジェクトの名称は[DNSサブドメイン名](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)として適切であり、かつ`system-`から始まってはいけません。
```
Comment (nasa9084) [inline content/ja/docs/concepts/configuration/pod-priority-preemption.md:52]:
```suggestion
PriorityClassオブジェクトの名称は[DNSサブドメイン名](/ja/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)として適切であり、かつ`system-`から始まってはいけません。
```

ditto (こちらはIDついているっぽいのでURL変更だけで大丈夫そうに見えます)

**Review by nasa9084 — CHANGES_REQUESTED** [review]

**Pull request merged**